### ISSUE TITLE
frontend: Link to deps.dev for supported packages

### ISSUE THREAD

**Issue by oliverchang** [google/osv.dev#1551]

**Comment by zahraaalizadeh** [comment]
@oliverchang Can you provide more details about this issue?

**Comment by another-rex** [comment]
On any particular vulnerability page (e.g. https://test.osv.dev/vulnerability/GHSA-7w47-3wg8-547c), we have the list of packages at the bottom, and the package name currently links to the default package manager page (e.g. https://crates.io/crates/gix-worktree-state). 

In addition to this link, it would be nice to have another link to the appropriate deps.dev page so people can learn more about the general information on the package. (e.g. https://deps.dev/cargo/gix-worktree-state)

**Comment by another-rex** [comment]
deps.dev does not support all the ecosystems that OSV.dev supports though (e.g. packagist/PHP is supported in OSV.dev, but not in deps.dev), so we'll need individual ecosystem configuration on whether to show a link there or not.

**Comment by andrewpollock** [comment]
I ❤️  the idea of boosting the visibility of the symbiotic relationship between OSV.dev and deps.dev.

Would it make sense to add another field (boolean, perhaps, or in the same vein as `human_link` in case the link is not guaranteed to be directly derived from OSV.dev's idea of the ecosystem name?) to https://github.com/google/osv.dev/blob/7ab7d213d4daba3f0c65954b06c035aad5feb498/osv/models.py#L790-L834 to enable this on a per-source basis?

**Comment by zahraaalizadeh** [comment]
Where is the best place to handle the mapping? Should it be done on frontend  at the time of the template rendering? or should the link be saved in the bug model?

**Comment by andrewpollock** [comment]
Comparing the ecosystem names in https://osv-vulnerabilities.storage.googleapis.com/ecosystems.txt with the current set deps.dev supports:

* https://deps.dev/cargo/openssl
* https://deps.dev/go/github.com%2Fspacemonkeygo%2Fopenssl
* https://deps.dev/maven/com.immomo.cosmos.mediax%3Aopenssl
* https://deps.dev/nuget/openssl-native
* https://deps.dev/pypi/opensslpy
* https://deps.dev/npm/openssl

Unsurprisingly there's going to need to be translation of the ecosystem name required, conveniently just lowercasing it, so it looks like it would be pretty straightforward to construct in the front end via a boolean for the supported ecosystems in `SourceRepository` model?

**Comment by zahraaalizadeh** [comment]
Sounds sensible to me. I have a couple of questions:

1. Adding New Fields and Backfilling Data: Is this something we can help with, or does it require backend work that might block us? If backfilling is needed, could you provide guidance on how we should approach this?

2. Temporary Frontend Solution: While we address the backend considerations, would it be acceptable to use a constant list of supported ecosystems on the frontend to unblock this issue for now?

**Comment by zahraaalizadeh** [comment]
As per @another-rex’s suggestion in today’s meeting, I will proceed with a temporary list of supported repositories for now.
